app/src/renderer/js/main.js

```diff
@@ -135,6 +136,10 @@ document.addEventListener('DOMContentLoaded', () => {
       }
     }
 
+    if (app.kap.settings.get('hideDesktopIcons')) {
+      await desktopIcons.hide();
+    }
+
```

Comment:
Maybe we should use something like http://npmjs.com/package/unload to show the icons if the app crashes during a recording (from here until it shows them again)? or will the child swift app die automatically? 🤔

Reply:
Yes it will? https://github.com/sindresorhus/execa#cleanup

Reply:
I think `child_process` dies if the parent dies (unless you use `detached`). So if the main process dies, I would expect the child one to die. When the script is killed, the swift window is closed.

I will test it tho and get back to you